[docs] remove deprecated versions, update SDK Upgrade page (#19962)
expo/expo: docs/pages/workflow/upgrading-expo-sdk-walkthrough.mdx

title: Upgrading Expo SDK
---

> If you are behind a couple of versions, upgrading your project's Expo SDK version can be difficult because of the number of breaking changes and deprecations in each upgrade.
> Don't worry. This document links to all the breaking changes in each SDK version upgrade.
> 

Reviewer: ```suggestion
```

> > **info** We **strongly recommend** upgrading SDK versions incrementally if possible. Doing so will help you pinpoint breakages and issues that arise during the upgrade process.

Reviewer: ```suggestion
> **info** We recommend upgrading SDK versions incrementally, one at a time. Doing so will help you pinpoint breakages and issues that arise during the upgrade process.
```
